### ISSUE TITLE
Add Google Colab direct links to the notebooks

### ISSUE THREAD

**Issue by mslavescu** [stephencwelch/self_driving_cars#5]
Excellent presentation of SDC algorithms!

Would be great if you could add Google Colab links for your Jupyter notebooks, like here:
https://medium.com/@mslavescu/try-live-ssd-object-detection-mask-r-cnn-object-detection-and-instance-segmentation-sfmlearner-df62bdc97d52
More details here:
You can run in Google Colab directly from GitHub ...
https://www.reddit.com/r/MachineLearning/comments/amfinl/project_nlptutoral_repository_who_is_studying/efmungz?utm_source=reddit-android

I'll be referring your videos and code about SDCs, tomorrow at this event "How about the future?" where I'll talk about past/present/future of SDCs:
https://www.linkedin.com/feed/update/urn:li:activity:6498896746742185984

**Comment by stephencwelch** [comment]
Yes, this would be nice to have. Hopefully I can find some time. Best of luck tomorrow - happy my videos can be of service!
